### PR TITLE
Serverless Training quickstart: train a support agent with Serverless RL

### PR DESCRIPTION
**Pull request by dbrian57**
## What this adds

The Serverless Training section previously had no in-house getting-started content — the "How to use" pages linked out to OpenPipe ART's external quickstart, whose example task (the 2048 game) is academic. This PR adds a self-contained quickstart built around one of the product's officially supported use cases: **customer support**.

- **`serverless-training/quickstart.mdx`** — trains a LoRA for a customer support agent (fictional smart thermostat) with Serverless RL: register `OpenPipe/Qwen3-14B-Instruct` with ART's `ServerlessBackend` → tool-calling rollout over an inline 10-article knowledge base → RULER scoring (no labeled data or hand-written reward function) → 6-step training loop → inference against the trained LoRA via the `wandb-artifact:///` endpoint.

### CODE REVIEW

**Comment by dbrian57** [comment]
## Sources and decision log

### Sources for factual claims
- **Product behavior, endpoints, pricing, W&B services used**: existing pages in this repo — `serverless-training.mdx`, `serverless-training/usage-limits.mdx`, `serverless-training/use-trained-models.mdx`, `serverless-training/available-models.mdx`, `serverless-training/api-reference.mdx` (base URL `https://api.training.wandb.ai/v1`, `wandb-artifact:///ENTITY/PROJECT/MODEL:STEP` endpoint schema, free training during preview).
- **ART API usage** (`TrainableModel`, `ServerlessBackend`, `model.register`, `model.inference_base_url` / `inference_api_key` / `get_inference_name`, `art.Trajectory`, `gather_trajectory_groups`, `iterate_dataset`, `backend.train`, `model.log`, `ruler_score_group`): OpenPipe's current ART·E notebook (`openpipe/art-notebooks/examples/art-e.ipynb`, pins `openpipe-art==0.5.9`) and art.openpipe.ai docs (art-client, installation-setup, ruler, sft-training pages).
- **Use-case selection** (customer support as an officially supported use case): internal launch messaging (Slack #revenue launch thread 2025-10-08; PMM guidance names voice agents, customer support, agentic RAG, deep research). Internal docs-plan context: Slack #serverless-training thread 2025-10-01 (launch docs intentionally barebones; eng lead open to new content); JIRA DOCS-2698 (Serverless Training umbrella naming/IA).
- **Base model choice**: `serverless-training/available-models.mdx` catalog (auto-generated).

### Key decisions
- **Customer support agent instead of adapting ART·E or 2048**: officially supported use case; self-contained inline knowledge base (no Enron dataset download); demonstrates the "no labeled data" RULER story. Confirmed with Dan Brian before drafting.
- **Full train→serve loop on one page**: internal launch threads agreed inference-with-LoRAs must be documented with training; the page ends at the `wandb-artifact` inference call and links to Use your trained models.
- **Manual tool JSON schema instead of `langchain_core.convert_to_openai_tool`**: drops three dependencies from the reader's environment vs. the ART·E notebook.
- **No validation split / checkpoint deletion in the loop**: kept the quickstart minimal; both are pointed to in Next steps (ART·E notebook, checkpoint-deletion docs).
- **No wall-clock/cost estimate**: intentionally omitted (confirmed with Dan) — no measured number exists for this exact configuration.

### Intentionally omitted
- Serverless SFT walkthrough (mentioned only in Next steps; candidate for a follow-up quickstart).
- Weave integration in the main code path (optional Tip instead, to keep required deps at `openpipe-art` only).

## Needs SME verification

- [ ] **End-to-end execution**: nobody has run the full training loop yet. Dan Brian will run the Colab; verify rollouts, RULER calls, and training steps complete against the current backend.
- [ ] **Model catalog freshness**: page uses `OpenPipe/Qwen3-14B-Instruct` from our published catalog, but OpenPipe's current ART·E notebook trains `Qwen/Qwen3.6-27B`, which isn't in `available-models.mdx` — is the auto-generated catalog current?
- [ ] **Checkpoint alias**: final example serves `...support-agent-001:step6`. Confirm a 6-step run labels its final checkpoint exactly `step6` (format inferred from `use-trained-models.mdx`).
- [ ] **RULER judge**: quickstart requires an OpenAI key for the judge (`openai/gpt-5.4`, matching OpenPipe's notebook). Is a W&B Inference-hosted judge supported through `ruler_score_group`? If so, a single-key story would onboard better (follow-up).
- [ ] **`iterate_dataset`/`backend.train` signatures**: match `openpipe-art` 0.5.9; confirm the pinned version customers get from a bare `pip install openpipe-art` hasn't drifted.

Suggested reviewers: David Corbitt (technical accuracy), John Mulhausen (docs IA/editorial).

## Resume prompt

> Continue work on wandb/docs PR #2879 (branch `dbrian/docs-serverless-training-quickstart`): the Serverless Training quickstart (`serverless-training/quickstart.mdx` + `serverless-training/cookbooks/train-support-agent.ipynb`). Decisions already made: customer support agent use case, full RL loop with RULER (OpenAI judge, `openai/gpt-5.4`), base model `OpenPipe/Qwen3-14B-Instruct`, page + Colab companion, no time/cost estimate. Remaining: (1) apply the style-guide pass files (bundled in the docs plugin skill directory) and push fixes; (2) incorporate results of Dan Brian's end-to-end Colab run, especially the actual checkpoint alias for step 6; (3) resolve the SME checklist above; (4) mark ready for review and assign David Corbitt + a TW reviewer. Note: the local repo's `css-tab-borders` branch has unrelated uncommitted CSS changes — use a git worktree for commits instead of switching branches.

```json
{
  "skill": "docs",
  "version": "unknown",
  "depends_on": "unknown",
  "note": "versions.json not present in the installed skill directory (cw-claude-code-plugins/docs/1.0.0/skills/docs); versions not guessed.",
  "model": "claude-fable-5"
}
```

**Comment by dbrian57** [comment]
Style-guide pass complete (commit 5bfa16677): descriptive headings without step numbering, bracket-style placeholders, complete-sentence list intros. Remaining before marking ready: end-to-end Colab run (Dan) and the SME checklist above.
